dev/com.ibm.ws.logging_fat/fat/src/com/ibm/ws/logging/fat/TestHideMessages.java

```diff
@@ -34,20 +35,29 @@ public class TestHideMessages {
 
     private static LibertyServer msgServer = LibertyServerFactory.getLibertyServer("com.ibm.ws.logging.hidemessage");
     private static final Class<?> logClass = TestHideMessages.class;
-    static String TWO_HIDDEDN_MESSAGE_SERVER = "server-twomessageids.xml";
+    static String TWO_HIDDEN_MESSAGE_SERVER = "server-twomessageids.xml";
```

Comment:
might as well fix the camel casing of the `server-twomessageids` aswell

Reply:
Fixed.